viewing_party/party.py

```diff
@@ -0,0 +1,171 @@
+#WAVE 1
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        movie = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    updated_data = user_data["watched"]
+    updated_data.append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    add_watchlist = user_data["watchlist"]
+    add_watchlist.append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    movies_watched = user_data["watched"]
+    movies_to_watch = user_data["watchlist"]
+    remove_list = []
+
+    for movie in movies_to_watch:
+        if title == movie["title"]:
+            movies_watched.append(movie)
+            remove_list.append(movie)   
+
+    for movie in remove_list:
+        movies_to_watch.remove(movie)
```

Comment:
I see what you did here! Definitely lowers the chances of side effects if you were to have `remove(movie)` in the previous for loop